msal4j-sdk/src/test/java/com/microsoft/aad/msal4j/TestHelper.java

```diff
@@ -97,4 +105,34 @@ static HttpResponse expectedResponse(int statusCode, String response) {
 
         return httpResponse;
     }
+
+    static void setPrivateKeyAndCert() {
```

Comment:
Can you use a static certificate instead? Like in MISE? That way you can assert on the the well known hash too.

Reply:
The variables are already static, or are you saying to just create one locally and add it to the repo?